hedera-node/test-clients/build.gradle.kts

```diff
@@ -77,6 +77,8 @@ val prCheckTags =
         put("hapiTestTimeConsuming", "LONG_RUNNING")
         put("hapiTestIss", "ISS")
         put("hapiTestBlockNodeCommunication", "BLOCK_NODE")
+        put("hapiTestClpr", "CLPR")
+        put("hapiTestMultiNetwork", "MULTINETWORK")
```

Comment:
Just a note so you're aware, we'll need to annotate any hapi tests that need to run in PR checks with `@Tag(MATS)`. The tests _should_ still run, but any failures would be triggered in the XTS suite instead of a PR gate